apps/protocol-ld/project.json

```diff
@@ -0,0 +1,51 @@
+{
+  "root": "apps/protocol-ld",
```

Comment:
We should document a naming convention, but I am thinking this should probably be called something like voyager-identity-api. Voyager Identity is the name of the metagov working froup